conductr_cli/sandbox_run_jvm.py

```diff
@@ -53,6 +52,7 @@ def run(args, features):
                      This is only relevant for Docker based sandbox since the features decides what port to expose
     :return: SandboxRunResult
     """
+    sandbox_common.LATEST_VERSION=args.image_version
```

Comment:
There's no constant called `LATEST_VERSION` declared in `sandbox_common`, but there's `LATEST_CONDUCTR_VERSION`.

I don't think we should be setting a constant like this (although `LATEST_VERSION` is a variable, but by convention that's it's all uppercase, it should be treated as a constant).

I don't think `LATEST_VERSION` is used anywhere, we shouldn't need to set them.